src/test/java/unit/Group.java

```diff
@@ -315,6 +294,32 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private void expectListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // call `terra group list-users` in table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
```

Comment:
If you want, you could change `TestCommand.runCommandExpectSuccess()` to return a `TestCommand.Result` instead of being `void` and then you wouldn't have to assert the stdErr and the exit code.

Reply:
I think we could not do it as void. because the header verification need the cmd.stdOut.